### DISCUSSION TITLE
Custom Filters View at Top of Table

### DISCUSSION THREAD

**Discussion by dhruva81** [rappasoft/laravel-livewire-tables#429] [Q&A]
Hi 
How can I show custom filter view on the top of Datatable similar to following image.

![image](https://user-images.githubusercontent.com/42173738/128357705-9202f828-5320-4a68-ae55-dcecec184f8f.png)

Kindly guide. 
Thank You

**Comment by rappasoft** [comment]
I wouldn't recommend it but you can disable the built in filters and modify the views to use your own.

**Reply by dhruva81** [reply]
Thank you for reply.
I have read documentation, but i am not able to find how to put this custom filter view on the top of data table.
Kindly guide.
Thank you

**Reply by rappasoft** [reply]
You would not use that one. You would disable the filters all together and publish and modify the views that come with the package. It would not be an easy task and there is no documentation on how to do that, but it is possible.

**Reply by dhruva81** [reply]
Okay.. 
Thank you for reply.
I am completely agree with you. It is not good to change any views coming with the package. 
Is there any way to add one custom filer slot in the package (for top position of table) that we can apply in component like

```
public function filtersAtTopLocation: string
{
    return 'path-to-view';
}

```
